.travis.yml

```diff
@@ -5,7 +5,7 @@ rvm:
 before_install:
   # Here, we decrypt a passwordless ssh key for acting as a user on the server named "travis"
   # See: https://docs.travis-ci.com/user/encrypting-files/#Automated-Encryption
-  - openssl aes-256-cbc -K $encrypted_a3d58fa41b45_key -iv $encrypted_a3d58fa41b45_iv -in scripts/id_rsa.enc -out scripts/id_rsa -d
+  - if [ "$TRAVIS_SECURE_ENV_VARS" == true ]; then openssl aes-256-cbc -K $encrypted_a3d58fa41b45_key -iv $encrypted_a3d58fa41b45_iv -in scripts/id_rsa.enc -out scripts/id_rsa -d; fi
   - chmod 400 scripts/id_rsa
```

Comment:
I think it will error without a wrap  around the chmod too, as the file won't exist
![screenshot](https://imgur.com/kBfphlP.png)